Gopkg.lock

```diff
@@ -84,6 +84,14 @@
   pruneopts = ""
   revision = "3a771d992973f24aa725d07868b467d1ddfceafb"
 
+[[projects]]
```

Comment:
When I run `make dep` against this branch, I get modifications to `Gopkg.lock`. Looks like it just reorders a few things.